ra-dec-positions.json

```diff
@@ -0,0 +1,12 @@
+    { "type": "record",
```

Comment:
This record is not used.

Reply:
This is the referenced schema that would be used in the future enhanced versions.